style.css

```
@@ -0,0 +1,71 @@
+header {
+	display: flex;
+	justify-content: flex-end;
+}
+a {
```

Краще використовувати класи і систему іменування
```suggestion
.navigation__link {
```